scipy/io/matlab/_mio5_utils.pyx

```diff
@@ -155,18 +153,18 @@ cdef class VarReader5:
         int squeeze_me
         int chars_as_strings
 
-    """ Initialize from file reader object
-
-    preader needs the following fields defined:
+    # Initialize from file reader object
+    #
+    # preader needs the following fields defined:
```

Comment:
Does not look right no?

Reply:
Could you explain?

Reply:
is this to get around the "pointless string statement" error? If you make this the docstring, then it won't be thrown, i.e.
```
cdef class VarReader5:
    """ Initialize from file reader object
    <rest of docstring>
    """
    cdef public int is_swapped, little_endian
```

(I may need to introduce configuration at some point..)